### PR TITLE
Adding integration test on message hub

### LINKED ISSUE #352: Integration Test - data-processing-app  (uses MessageHub)
Create an integration test for validating [data-processing-app](https://github.com/apache/incubator-openwhisk-wskdeploy/tree/master/tests/usecases/data-processing-app) usecase.

Integration tests are located under tests/src/integration. Create a new folder named "data-processing-app" under tests/src/integration.

Remove data-processing-app from usecase folder.

### LINKED ISSUE #474: WIP - Use Case - message hub
Please add a new use case for Kafka message hub, we don't have one (like cloudant), showing how to write manifest/deployment with message hub in our repo. Please add a realistic use case demonstrating how to configure message hub with its api key if possible.

### PR DESCRIPTION
**Pull request by pritidesai**
Closes #474 
Closes #352 

Fails with arbitrary errors:

```
==> Error: Error: /src/github.com/apache/incubator-openwhisk-wskdeploy/deployers/servicedeployer.go [554]: OpenWhisk Client Error =====> The following application error was received: {} Error code: 246.
```

```
==> Error: Error: /src/github.com/apache/incubator-openwhisk-wskdeploy/deployers/servicedeployer.go [914]: OpenWhisk Client Error =====> [object Object] Error code: 246.
```

### CODE REVIEW

File: tests/src/integration/message-hub/message-hub_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/* *
+ * Please configure following env. variables in order to run this integration test:
+ * 	BLUEMIX_APIHOST
+ *	BLUEMIX_NAMESPACE
+ * 	BLUEMIX_AUTH
+ *	MESSAGEHUB_USER
+ *	MESSAGEHUB_PASSWORD
+ */
+func TestAlarmTrigger(t *testing.T) {
```
Comment (houshengbo) [inline tests/src/integration/message-hub/message-hub_test.go]: Please change the func name.

File: tests/src/integration/message-hub/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+packages:
+    data-processing-with-messagehub:
+        dependencies:
+            wskdeployMessageHub:
+                location: /whisk.system/messaging
+                inputs:
+                    user: $MESSAGEHUB_USER
+                    password: $MESSAGEHUB_PASSWORD
+                    kafka_admin_url: $MESSAGEHUB_ADMIN_HOST
```
Comment (houshengbo) [inline tests/src/integration/message-hub/manifest.yaml]: How can we pick up the admin host in Travis?

File: tests/src/integration/message-hub/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                    kafka_brokers_sasl: $KAFKA_BROKERS_SASL
```
Comment (houshengbo) [inline tests/src/integration/message-hub/manifest.yaml]: How can we pick up the brokers in Travis?

File: tests/src/integration/message-hub/manifest.yaml (hunk 2 above)
Comment (pritidesai) [reply tests/src/integration/message-hub/manifest.yaml]: I am setting it in the test file `message-hub_test.go`

File: tests/src/integration/message-hub/manifest.yaml (hunk 3 above)
Comment (pritidesai) [reply tests/src/integration/message-hub/manifest.yaml]: setting it in `message-hub_test.go`

File: tests/src/integration/message-hub/manifest.yaml (hunk 2 above)
Comment (pritidesai) [reply tests/src/integration/message-hub/manifest.yaml]: https://github.com/apache/incubator-openwhisk-wskdeploy/pull/528/files#diff-189c157cd9e283389d43479ba4b0cd86R39

File: tests/src/integration/message-hub/manifest.yaml (hunk 3 above)
Comment (pritidesai) [reply tests/src/integration/message-hub/manifest.yaml]: https://github.com/apache/incubator-openwhisk-wskdeploy/pull/528/files#diff-189c157cd9e283389d43479ba4b0cd86R40

**Comment by houshengbo** [comment]
@pritidesai @mrutkows I tried to create the trigger with the message hub feed this morning with CLI command, and it turned out fine. 
However, when I tried to use the same params to run the integration test. IT FAILED TO DEPLOY, and of course if failed to undeploy too. It failed when to invoke the feed action with params we provide.

I am digging into the issue why it failed to create the feed action.

**Pull request merged**